packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -39,6 +39,14 @@ double _getCssOpacity(Color color) {
   return color.opacity;
 }
 
+// Converts a [Color] into a valid CSS value rgba(R, G, B, A).
+String _getCssColorWithAlpha(Color color) {
```

Comment:
I wonder if we can change the strokeColor + strokeOpacity combo in the geometry drawing (Circles/Polys) by passing a `rgba` color directly to strokeColor.

[Docs](https://developers.google.com/maps/documentation/javascript/reference/3.46/polygon#PolylineOptions.strokeColor) say that _"All CSS3 colors are supported"_ as `strokeColor`.

Reply:
Is this within the scope of this PR?

Reply:
Nah, I'll experiment with it once you land this :)